blog-server-services/src/impls/rbatis_post_service.rs

```diff
@@ -1,13 +1,32 @@
-use crate::traits::post_service::{BasePost, Post, PostService};
+use crate::traits::post_service::{BasePost, Post, PostService, Tag};
 use rbatis::rbatis::RBatis;
 use screw_components::dyn_result::{DError, DResult};
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
 
 pub fn create_rbatis_post_service(rb: RBatis) -> Box<dyn PostService> {
     Box::new(RbatisPostService { rb })
 }
 
 impl_insert!(BasePost {}, "post");
 
+#[derive(Clone, Debug, Serialize, Deserialize)]
+#[serde(rename_all = "snake_case")]
+pub struct TagDto {
```

Comment:
```suggestion
struct TagDto {
```

Reply:
![image](https://github.com/tikitko/blog-server/assets/52403678/7c56938e-26f0-407b-b5ff-62606ab81e82)

looks like its better to let it be public

Reply:
BAZA